pkgs/servers/invidious/update.sh

```diff
@@ -60,7 +60,7 @@ info "Running scripts/fetch-player-dependencies.cr..."
 git -C "$git_dir" reset --hard "$new_rev"
 (cd "$git_dir" && crystal run scripts/fetch-player-dependencies.cr -- --minified)
 rm -f "$git_dir/assets/videojs/.gitignore"
-videojs_new_sha256=$(nix hash-path --type sha256 --base32 "$git_dir/assets/videojs")
+videojs_new_sha256=$(nix --extra-experimental-features nix-command hash path --type sha256 --base32 "$git_dir/assets/videojs")
```

Comment:
```suggestion
videojs_new_sha256=$(nix-hash --type sha256 --base32 "$git_dir/assets/videojs")
```

Let's use stable commands when possible